.circleci/config.yml

```diff
@@ -0,0 +1,85 @@
+version: 2.1
+
+aliases:
+    - &root ~/radiance-ui
+
+    - &defaults
+        working_directory: *root
+
+    - &cache-yarn-key-base yarn-packages-v1-{{ arch }}-
+    - &cache-yarn-key yarn-packages-v1-{{ arch }}-{{ checksum "yarn.lock" }}
+
+    - &yarn-cache-restore
+        restore_cache:
+            keys:
+                - *cache-yarn-key
+                - *cache-yarn-key-base
+
+    - &yarn-cache-save
+        save_cache:
+            key: *cache-yarn-key
+            paths:
+                - ~/.cache/yarn
+
+    # Prospective merging against master.
+    - &prospective-merge
+        run:
+            name: Prospective Merge
+            command: |
+                if [[ ! -z "${CIRCLE_PULL_REQUEST}" ]]; then
+                    git pull --ff-only origin "refs/pull/${CIRCLE_PULL_REQUEST//*pull\//}/merge"
+                fi
+
+jobs:
+    build:
+        <<: *defaults
+
+        docker:
+            - image: circleci/node:10-stretch
+
+        steps:
+            - checkout
+            - <<: *prospective-merge
+            - <<: *yarn-cache-restore
+            - run:
+                name: Install and Build
```

Comment:
Based on the [Codeship project](https://app.codeship.com/projects/312533/tests/edit) it seems like we only need to invoke `yarn install`. I would probably change this to `Install` and remove the `yarn build` invocation.